### ISSUE TITLE
debug atom beautify error

### ISSUE THREAD

**Issue by crazypattern** [Glavin001/atom-beautify#1272]
# Description

The results of beautification are not what I expect.
# Expected Results

The beautified code should have looked like:

```
<INSERT CODE HERE>
```
# Steps to Reproduce
1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!
# Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/53de27e97b446388c20e86e87bf31f2c
# Checklist
- [ ] I have tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [ ] I have reloaded (or restarted) Atom to ensure it is not a caching issue
- [ ] Searched for existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
  so I know this is not a duplicate issue
- [ ] Generated debugging information and added link for `debug.md` Gist to this issue

**Comment by prettydiff** [comment]
I dropped the code sample into http://jsbeautifier.org/ and I got broken HTML.  Alerting @bitwiseman

**Comment by bitwiseman** [comment]
@prettydiff 
If the output your are seeing matches the debug, this is another example of someone not turning on `e4x` for a jsx input.   

Also, I'm extremely glad that you have setup the `debug.md` function and I deeply appreciate your looping me in on issues that relate to the beautifier, but I really need clear and concise bug reports to understand and take action on issues.   Input, expected, and output. 

Tagging me on an issue that hasn't been fully analyzed just makes me use the limit bandwidth I have analyzing issues instead of fixing them.

**Comment by Glavin001** [comment]
>  I really need clear and concise bug reports to understand and take action on issues. Input, expected, and output.
>
> Tagging me on an issue that hasn't been fully analyzed just makes me use the limit bandwidth I have analyzing issues instead of fixing them.

I absolutely agree, @bitwiseman. We need to put our foot down on Issue report quality. If an issue is not a clear, relatively small example with input and output (from `debug.md`), and expected (often missing) then we cannot spend time analyzing it when other users have properly reported their Issue.

@prettydiff let's use the `waiting-for-user-information` label more often when applicable and only once we have the information let's begin the analysis ourselves, and/or call in @bitwiseman for assistance. We have hundreds of Issues and limited bandwidth.

On a related note, thank you to both @prettydiff and @bitwiseman for your support!

**Comment by Glavin001** [comment]
>  this is another example of someone not turning on e4x for a jsx input.

I'm closing this as a duplicate of https://github.com/Glavin001/atom-beautify/issues/1165#issuecomment-269891806 which is to add JSX support.
